terra/contracts/cw721-base/Cargo.toml

```diff
@@ -0,0 +1,38 @@
+[package]
```

Comment:
I wrote the reason in the commit message:

https://github.com/certusone/wormhole/commit/0a5c6f885349fce5337647034ce146ac944ceed5

though it would make sense to record this information in the repo itself, which I will do.

Reply:
Any reason you chose to not override their sdk version? (https://doc.rust-lang.org/cargo/reference/overriding-dependencies.html)

Reply:
I looked at the patch feature of cargo, but to me it seemed that it only allows overriding the package registry, but not overriding the version bounds of upstream dependencies.

Reply:
Makes sense, I thought you could also fix the version but it seems like that's indeed not possible